framework/src/main/java/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -1418,6 +1419,11 @@ private Object getValueOfExpressionInAnnotation(
             return convert(((IntegerLiteralExpr) expr).asInt(), valueKind);
         } else if (expr instanceof LongLiteralExpr) {
             return convert(((LongLiteralExpr) expr).asLong(), valueKind);
+        } else if (expr instanceof UnaryExpr) {
+            if (((UnaryExpr) expr).getOperator() == UnaryExpr.Operator.MINUS) {
+                int value = Integer.parseInt(expr.toString());
```

Comment:
What if the negated integer is a long rather than an int?

Reply:
Addressed